libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/GenState.hs

```diff
@@ -93,74 +98,214 @@ import Test.Tasty.QuickCheck
     generate,
   )
 
+-- import Debug.Trace(trace)
+
 -- =================================================
 
 -- | Constants that determine how big a GenState is generated.
 data GenSize = GenSize
-  { numKeys :: Int,
-    numScripts :: Int,
-    numDatums :: Int,
-    numRewards :: Int,
-    numPools :: Int
+  { treasury :: Integer, -- DO WE EVER USE THIS?
```

Comment:
For consistency sake with Haskell conventions and the rest of the repo (at least hopefully the ultimate direction of it) prefixing all record fields with common prefix, eg: `gsTreasury` or `gSizeTreasury`? That comment naturally applies to all fields of this type